app/components/candidate_interface/application_complete_content_component.html.erb

```diff
@@ -19,6 +19,10 @@
   <h2 class="govuk-heading-m govuk-!-font-size-27">
     <%= t('application_complete.dashboard.all_withdrawn') %>
   </h2>
+<% elsif application_not_sent? %>
```

Comment:
This won't show if one of the choices is cancelled or withdrawn. Is that the intended behaviour?

Reply:
Good point. Have fixed.